homework02.py

```diff
@@ -0,0 +1,17 @@
+text = input("Write word or number:")
+if text.isdigit():
+if int(text) % 2 == 0:
+        print("This is a number")
+        print("Number is even")
+    # ===========================================#
+
+elif text.isdigit():
```

Comment:
You can have only one check for .isdigit() and have a few nested conditions, like this:
```
if text.isdigit():
    if <some other condigion>:
        # do something
    else:
        # do something else
else:
    # do something with not a digit